src/building/type.h

```diff
@@ -10,6 +10,7 @@
   * Building types
   */
 typedef enum {
+    BUILDING_ANY = -1,
```

Comment:
I think you should set `BUILDING_ANY` to `0`, not `-1`. You can duplicate references to the same value in an enum.

Reply:
How would I then distinguish between the two when I load the value from file?
I.e. of the saved data of the event only shows a 0, how do I know if it was meant to check for any or none?

Reply:
Would you ever want to check for none, though? You can check for any and if one is found, you know a building shouldn't be there.

Reply:
That is a good point.

Reply:
Done.